packs/vsphere/actions/vmwarelib/inventory.py

```diff
@@ -1,23 +1,9 @@
-# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
the version I originally worked from must have not had it.. but i'll get it added back in.